File: tests/javascript/sample.js
const MAX = 100;
export const NAME = "app";

export class Greeter {
  constructor(name) {
    this.name = name;
  }
  greet() {
    return `hi ${this.name}`;
  }
  async load() {}
}

function add(a, b) {
  // function notReal() in a comment
  return a + b;
}

const mul = (a, b) => a * b;

File: tests/ruby/sample.rb
MAX = 10

class Greeter
  GREETING = "hi"

  def initialize(name)
    @name = name
  end

  def greet
    "#{GREETING} #{@name}"
  end
end

module Helpers
  def self.run
  end
end

def add(a, b)
  a + b
end
